ps3t_game_list.php

```diff
@@ -161,21 +174,22 @@
                     //		$nextpage = 1; } 
                     //}
                     //else {
-                    if ($x->href == '/browsegames/' . $gametype[$i] . '/' . $gamealpha . '/' . $k) {
-                        $nextpage = 1; }	
+                    //if ($x->href == '/browsegames/' . $gametype[$i] . '/' . $gamealpha . '/' . $k) {
+                    //    $nextpage = 1; }	
                     //}
                     
                 }
                 
-            }
+            //}
             
         }
         
         echo '<BR>';
         
         $html->clear(); 
         unset($html);
-        
+        $init_html->clear();
+        unset($init_html);

```

Comment:
Let's put this clear/unset before line 112... once you get $page_num you don't need to retain the object for the rest of the script.

Reply:
Makes sense... smart!